packages/ui-kit/src/components/InfoTip/InfoTip.tsx

```diff
@@ -7,22 +7,22 @@ import { Text } from "../Text";
 export interface InfoTipProps {
   content: string;
   className?: string;
-  indicatorPosition?: "left" | "right";
+  alignment?: "left" | "right";
```

Comment:
I think I mentioned somewhere putting this a type e.g `HorizontalAlignment`?

Reply:
Done.